Tester/test_fillerwithvaluesfromtxtfile.cpp

```diff
@@ -0,0 +1,20 @@
+#include "Tester_Catch.h"
+#include "../GameOfLife/fillerwithvaluesfromtxtfile.h"
+#include <string>
+#include "../GameOfLife/board.h"
+#include "../GameOfLife/sizewithframe.h"
+
+TEST_CASE("check if filler set values with given by string", "[tes_validator]")
+{
+    FillerWithValuesFromTxtFile filler;
+    std::string givenString = "X \n X";
+    SizeWithFrame size(2,2);
+    Board board(size);
+    filler.setBoard(board);
+    filler.getStringFromTxtFileReader(givenString);
```

Comment:
getStringFromTxtFileReader - ??
This name indicates, that the method takes TxtFileReader as a parameter and retunrns string.